### PR TITLE
feat: add dev helm 

### PR DESCRIPTION
**Pull request by hi-rustin**

### CODE REVIEW

**Comment by hi-rustin** [comment]
/assign @Mini256 

PTAL~ Thanks!

File: deployments/helms/dev/values.yaml [hunk 1]
```diff
@@ -0,0 +1,116 @@
+affiliations: 'github_users.json'
```
Comment (Mini256) [inline deployments/helms/dev/values.yaml:1]: Perhaps it would be better to sort the configuration items in the configuration file, and add some comments.

Comment (hi-rustin) [reply deployments/helms/dev/values.yaml:1]: I think it should be difficult to sort it, is there any benefit to doing so? But I think the related configurations are better put together. About writing comments, maybe we need a better documentation for these configurations.

Comment (hi-rustin) [reply deployments/helms/dev/values.yaml:1]: >  we need a better documentation for these configurations.

I don't think it's needed right now. There are actually many configurations that we are not sure how to use and may need to be added gradually.

Comment (Mini256) [reply deployments/helms/dev/values.yaml:1]: Yes, what I means is  put the related configurations to be together.

Comment (hi-rustin) [reply deployments/helms/dev/values.yaml:1]: I think most of the content is already categorized.

Comment (Mini256) [reply deployments/helms/dev/values.yaml:1]: Some common configurations can be put on the front, like this:
image

Comment (hi-rustin) [reply deployments/helms/dev/values.yaml:15]: Fixed.

File: deployments/helms/dev/values.yaml [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+syncRestartPolicy: Never
+indexCronsFrom: 0
+indexCronsTo: 100
+limitsCronsCPU: '6000m'
+limitsCronsMemory: '16Gi'
+requestsCronsCPU: '1000m'
+requestsCronsMemory: '2Gi'
+useCronsResourcesLimits: 1
+
+# Ingresses indexIngressesFrom.
```
Comment (Mini256) [inline deployments/helms/dev/values.yaml:92]:
```suggestion
# Ingresses indexes.
```

**Pull request merged**